src/Conversation.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * This file is part of the TelegramBot package.
+ *
+ * (c) Avtandil Kikabidze aka LONGMAN <akalongman@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Longman\TelegramBot;
+
+use Longman\TelegramBot\Command;
+use Longman\TelegramBot\Request;
+use Longman\TelegramBot\ConversationDB;
+use Longman\TelegramBot\Entities\Update;
+
+/**
+ * Class Conversation
+ */
+class Conversation
+{
+    /**
+     * Conversation has been fetched true false
+     *
+     * @var bool
+     */
+    protected $is_fetched = false;
+
+    /**
+     * All information fetched from the database
+     *
+     * @var array
+     */
+    protected $conversation = null;
+
+    /**
+     * Data stored inside the Conversation
+     *
+     * @var array
+     */
+    protected $data = null;
+
+    /**
+     * Telegram user id
+     *
+     * @var int
+     */
+    protected $user_id;
+
+    /**
+     * Telegram chat_id
+     *
+     * @var int
+     */
+    protected $chat_id;
+
+    /**
+     * Group name let you share the session among commands
+     * Call this as the same name of the command if you don't need to share the conversation
+     *
+     * @var strint
+     */
+    protected $group_name;
+
+    /**
+     * Command to be execute if the conversation is active
+     *
+     * @var string
+     */
+    protected $command;
+
+    /**
+     * Conversation contructor initialize a new conversation
+     *
+     * @param int    $user_id
+     * @param int    $chat_id
+     * @param string $name
+     * @param string $command
+     */
+    public function __construct($user_id, $chat_id, $group_name = null, $command = null)
+    {
+        if (is_null($command)) {
+            $command = $group_name;
+        }
+
+        $this->user_id = $user_id;
+        $this->chat_id = $chat_id;
+        $this->command = $command;
+        $this->group_name = $group_name;
+    }
+
+    /**
+     * Check if the conversation already exist
+     *
+     * @return bool
+     */
+    protected function conversationExist()
```

Comment:
Much better :+1: 
Using **exists** instead of **exist** is easier to understand linguistically.
In English you would ask "if something exists", so `$this->exists()` or `$that->exists()` is easier to understand :blush: